src/data.js

```diff
@@ -52,3 +61,27 @@ window.editTodo = (i, b) => {
   }
   window.location.reload();
 };
+
+window.changed = (i, b) => {
+  const change = JSON.parse(localStorage.getItem('todoLocal'));
+  for (let i = 0; i < change.length; i += 1) {
+    if (todo.todoArr[i].description === b) {
+      todo.todoArr[i].completed = true;
+      localStorage.setItem('todoLocal', JSON.stringify(todo.todoArr));
+    }
+  }
+  window.location.reload();
```

Comment:
- [ ] _OPTIONAL_ I would suggest learning about html data attributes and JS hashmaps, try to combine them to achieve this behavior instead of reload the view.